src/main/java/me/totalfreedom/totalfreedommod/caging/Cager.java

```diff
@@ -96,6 +97,18 @@ public void onPlayerQuit(PlayerQuitEvent event)
     }
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+    public void onPlayerKick(PlayerKickEvent event)
```

Comment:
yeah but ignoreCancelled is true, causing it to ignore the event being cancelled.